source/blog/2016-05-19-introducing-subteams.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Introducing Subteams
+author: Tom Dale
+tags: Recent Posts
+---
+
+I'm very happy to share with you our plan to scale Ember's day-to-day
+decision-making and coordination–something we're calling Subteams.
+
+Ember.js was once just a view layer for rendering templates in the
+browser, but has grown to become a complete SDK for the web. With one
+`npm install`, you get everything you need to create a modern web
+application.
+
+Making things simple for developers often means a lot of coordination
+and decision-making behind the scenes. As the number of pieces making up
+Ember has grown—encompassing not only Ember.js but Ember CLI, Ember
+Data, the Ember Inspector, documentation, and more—we want to ensure
+that the core team does not become a bottleneck for progress.
+
+At the same time, we believe that having a strong vision that everyone
+can rally behind is critical to building software that feels cohesive.
+So how do we ensure that everyone is pulling their cart in the same
+direction, without losing momentum?
+
+Today, we're taking a page from [Rust's
```

Comment:
As mentioned at EmberConf? Seems like it's already been happening for a bit, just wasn't announced here ...